examples/solidstart/.gitignore

```diff
@@ -2,6 +2,7 @@ dist
 worker
 .solid
 .vercel
+.output
```

Comment:
Why does the example need to ignore .output

Seems like we should use `.vercel/output` instead which is already covered by the ignore of `.vercel`

Reply:
SolidStart actually outputs FS API v2. That's the main purpose of this PR.